modular/code/game/objects/pillory.dm

```diff
@@ -142,7 +142,11 @@
 		to_chat(usr, span_warning("It doesn't look like [M.p_they()] can fit into this properly!"))
 		return FALSE // Can't hold non-humanoids
 
-	return ..(M, force, FALSE)
+	for(var/obj/item/grabbing/G in M.grabbedby)
+		if(G.grab_state == 1)
+			return ..(M, force, FALSE)
+	to_chat(usr, span_warning("You must grab them more forcefully to put them in the pillory."))
```

Comment:
Can you please change this warning to be in first person?

It's how most of these are done!

Reply:
Sure